lib/UUID4/Tiny.pm

```diff
@@ -55,7 +57,8 @@ sub string_to_uuid {
         carp 'Input not converted: assumed to be UUID bytes';
         return $string;
     }
-    pack 'H*', $string =~ y/-//dr;
+    (my $hex = $string) =~ s/-//g;
```

Comment:
We can keep it a tr though right? We don't need the flexibility of the regex engine.

Reply:
Switched it back sans `r`